### DISCUSSION TITLE
Pass data between Ops in-memory while using the SingleExecutor and rest of asset steps use k8s executor

### DISCUSSION THREAD

**Discussion by ion-elgreco** [dagster-io/dagster#19587] [Q&A] [answered]
Is it possible to pass data between Ops steps in a single execution context in memory, while the rest of the steps that are assets are materializing to the storage layer using k8s executor?

If we should serialize between the Ops steps it kind of ruins what I think the Ops makes itself useful and that's to chain certain operations and keep the lineage of it while also doing them fast. Having to serialize huge datasets to disk and then reading it back is very costly in storage and slower, so I am hoping there is a way around this :)

**Comment by ion-elgreco** [comment]
Having digged through issues and docs this seems to not be possible

**Comment by garethbrickman** [comment, marked as answer]
I see you re-worked this into a feature request Issue: https://github.com/dagster-io/dagster/issues/19589 Thanks so much!
